go_metrics/metrics/graphite.py

```diff
@@ -46,9 +51,21 @@ def zeroize_nulls(datapoints):
 
 
 class GraphiteMetrics(Metrics):
+    aggregators = {
+        'sum': SUM,
+        'avg': AVG,
+        'max': MAX,
+        'min': MIN,
+        'last': LAST,
+    }
+
+    def _get_full_metric_name(self, name):
+        return '%s.campaigns.%s.%s' % (
```

Comment:
I've been wondering whether we should make the `.campaigns.` part of the prefix and then metric names become completely controllable and the full name is just `"%s.%s.%s" % (prefix, owner, metric_name)`?